go.mod

```
@@ -5,15 +5,15 @@ go 1.16
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/fatih/structtag v1.1.0
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v1.2.0
 	github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
 	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334
 	github.com/kr/text v0.2.0
 	github.com/markbates/inflect v1.0.4
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.2.2
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.15.0
-	github.com/operator-framework/api v0.11.1
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.17.0
+	github.com/operator-framework/api v0.11.2-0.20220117134919-77b398430e01
 	github.com/operator-framework/java-operator-plugins v0.1.0
 	github.com/operator-framework/operator-lib v0.6.0
```

```suggestion
	github.com/operator-framework/operator-lib v0.6.0
```
An operator-lib bump maybe needed for this. Ill cut a release as soon as the issue related to CI is figured here: https://github.com/operator-framework/operator-lib/pull/98